crm_claim_ref_smartbutton/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# (c) 2015 Pedro M. Baeza
+# License AGPL-3 - See LICENSE file on root folder for details
+##############################################################################
+
+{
+    "name": "Smart-button for referenced claims",
+    "version": "1.0",
+    "depends": [
+        "crm_claim",
+    ],
+    "author": "AvanzOSC, "
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/crm",
+    "contributors": [
+        "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
+        "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
+    ],
+    "category": "Customer Relationship Management",
+    "installable": True,
+    "pre_init_hook": "pre_init_hook",
```

Comment:
@pedrobaeza  hooks are only called at installation time not each time the server start. Therefore you monkey-patch will be only applied once. IOW your patch will no more be applied after the first time your sever will be restarted after the installation. https://github.com/odoo/odoo/commit/4105b5f0285821d8056c721b5fd50b9cae902988#diff-9f65b6631806b04e785c2acdf3854285R151